tests/test_vehicle.py

```diff
@@ -3,9 +3,8 @@
 
 def test_create_vehicle(setup, vehicle):
     assert len(vehicle) == 8
-    assert isinstance(vehicle.get("year"), int)
     (vehicle_values_type,) = set([type(value).__name__ for value in vehicle.values()][1:])
-    assert vehicle_values_type  == "str"
+    assert vehicle_values_type == "str"
```

Comment:
or `is str`?

Reply:
I tried out `assert vehicle_values_type is str` but failed

<img width="743" alt="Screenshot 2023-12-19 at 4 35 38 PM" src="https://github.com/BenGriffith/tollway-traffic/assets/6089155/950a8650-0b63-4127-85de-fcf23a15c8ac">

Reply:
ah ok it's literally the string 'str' then, ok